language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -639,69 +639,87 @@ describe("interface definition", () => {
       expect(c).not.toEqual(theChoice(if2[k]));
       expect(c.template()).toBe(tpl);
     });
+
+    test("retroactive interfaces permit contract ID conversion", () => {
```

Comment:
any reason we're not testing exercises somewhere?

Reply:
The tests further down test interface exercise; there is absolutely no difference for retros from the perspective of codegen.